ipa-core/src/bin/report_collector.rs

```diff
@@ -329,6 +342,55 @@ fn write_ipa_output_file(
     Ok(())
 }
 
+async fn hybrid(
+    args: &Args,
+    hybrid_query_config: HybridQueryParams,
+    helper_clients: &[IpaHttpClient<Helper>; 3],
```

Comment:
sorry @akoshelev should have made a comment here directly. I'm not sure how this needs to be updated. it doesn't seem like oprf_ipa uses https (even for malicious). additionally, this needs to be 3 * number of shards, so presumably needs to be a vec, not an array.

Reply:
eventual consistency is fun. Yea, we need at 3*number of shards clients and sharded shuffle uses that currently. As for `https` we seem to use [it](https://github.com/akoshelev/raw-ipa/blob/cf7efd485096b3132e44366a6c0aad7d263ecd21/ipa-core/src/bin/report_collector.rs#L172)

Reply:
agreed that we need to use `https` scheme, but I'm not seeing an `IpaHttpsClient` struct, so this specific type may not need to change. I'll take a look at sharded shuffle to try and sort it out.